### PR TITLE
fixes #37 |  Update documents  hyperlinks

### LINKED ISSUE #37: Update Contributing document

### PR DESCRIPTION
**Pull request by rowlandekemezie**
This PR updates #53 because some hyperlinks weren't working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,10 +47,10 @@ gulp
 ```
 
 ## Contributing
-Refer to [Contibuting Docuemnent]('./docs/CONTRIBUTING.MD) to contribute to this project.
+Refer to [Contibuting Docuemnent](./docs/CONTRIBUTING.MD) to contribute to this project.
```
Comment (ghost) [inline README.md]: Typo error in`[Contibuting Document]`. Meant to be `[Contributing Document]`. @andela-rekemezie

**Review by ghost — CHANGES_REQUESTED** [review]
Some typo errors on line 50 in the README.md file.

File: README.md (hunk 1 above)
Comment (rowlandekemezie) [reply README.md]: Thanks! 
Fixed

**Comment by a-okelawal** [comment]
What do we need a contribution.md and a waffle.md document for?

**Comment by rowlandekemezie** [comment]
@Saint-lawal To help people contribute to the project. Check develop branch and see it's usefulness. In the meantime, approve the PR if you can.

File: README.md (hunk 1 above)
Comment (rowlandekemezie) [reply README.md]: @adetolaraphael the changes you requested has been fixed. Simply approved or request for more changes and I'd merge the issue.

**Review by ghost — APPROVED** [review]

**Pull request merged**